arangod/RestHandler/RestMetricsHandler.cpp

```diff
@@ -148,16 +130,27 @@ RestStatus RestMetricsHandler::execute() {
 
   auto& metrics = server().getFeature<metrics::MetricsFeature>();
   if (!metrics.exportAPI()) {
-    // dont export metrics, if so desired
+    // don't export metrics, if so desired
     generateError(rest::ResponseCode::NOT_FOUND, TRI_ERROR_HTTP_NOT_FOUND);
     return RestStatus::DONE;
   }
 
-  std::string result;
-  metrics.toPrometheus(result);
-  _response->setResponseCode(rest::ResponseCode::OK);
-  _response->setContentType(rest::ContentType::TEXT);
-  _response->addRawPayload(result);
-
+  auto const& values = _request->values();
+  auto it = values.find("json");
```

Comment:
TBH I'd be in favor of using `type`instead of just a marker so that we can easily extend it in future